### ISSUE TITLE
Local Dependencies not found while install dependencies

### ISSUE THREAD

**Issue by Inoir** [cloudfoundry/nodejs-buildpack#137]
_What version of Cloud Foundry and CF CLI are you using? (i.e. What is the output of running `cf curl /v2/info && cf version`?_
api_version: 2.65.0
cf-version: 6.25.0+787326d95.2017-02-28

_What version of the buildpack you are using?_
1.6.21 (1.6.20 works fine)
Node: 8.9.4
NPM: 5.6.0

_What did you expect to happen?_
Local Dependencies getting installed like before

_What was the actual behavior?_
Local Dependencies not found.
ErrorMessage: ENOENT

Please confirm where necessary:
* [X] I have included a log output
* [X] My log includes an error message
* [ ] I have included steps for reproduction

**LOG:**
```
2018-04-10T18:04:21.57+0200 [STG/0]      OUT Staging...
2018-04-10T18:04:25.17+0200 [STG/0]      OUT -----> Download go 1.9.1
2018-04-10T18:04:28.43+0200 [STG/0]      OUT -----> Running go build supply
2018-04-10T18:04:30.12+0200 [STG/0]      OUT -----> Nodejs Buildpack version 1.6.21
2018-04-10T18:04:30.20+0200 [STG/0]      OUT -----> Installing binaries
2018-04-10T18:04:30.20+0200 [STG/0]      OUT        engines.node (package.json): 8.9.4
2018-04-10T18:04:30.20+0200 [STG/0]      OUT        engines.npm (package.json): unspecified(use default)
2018-04-10T18:04:30.20+0200 [STG/0]      OUT -----> Installing node 8.9.4
2018-04-10T18:04:30.20+0200 [STG/0]      OUT        Download [https://buildpacks.cloudfoundry.org/dependencies/node/node-8.9.4-linux-x64-40e8e080.tgz]
2018-04-10T18:04:32.25+0200 [STG/0]      OUT        Using default npm version: 5.6.0
2018-04-10T18:04:32.25+0200 [STG/0]      OUT -----> Installing yarn 1.5.1
2018-04-10T18:04:32.25+0200 [STG/0]      OUT        Download [https://buildpacks.cloudfoundry.org/dependencies/yarn/yarn-v1.5.1-cd316572.tar.gz]
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        Installed yarn 1.5.1
2018-04-10T18:04:32.74+0200 [STG/0]      OUT -----> Creating runtime environment
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        PRO TIP: It is recommended to vendor the application's Node.js dependencies
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        Visit http://docs.cloudfoundry.org/buildpacks/node/index.html#vendoring
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NODE_ENV=production
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NODE_HOME=/tmp/app/.cloudfoundry/0/node
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NODE_MODULES_CACHE=true
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NODE_VERBOSE=false
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NPM_CONFIG_LOGLEVEL=error
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        NPM_CONFIG_PRODUCTION=true
2018-04-10T18:04:32.74+0200 [STG/0]      OUT -----> Building dependencies
2018-04-10T18:04:32.74+0200 [STG/0]      OUT        Installing node modules (package.json)
2018-04-10T18:04:34.51+0200 [STG/0]      OUT npm ERR! path /tmp/app/.cloudfoundry/0/packages/local_modules/api-1.0.0.tgz
2018-04-10T18:04:34.51+0200 [STG/0]      OUT npm ERR! code ENOENT
2018-04-10T18:04:34.51+0200 [STG/0]      OUT npm ERR! errno -2
2018-04-10T18:04:34.51+0200 [STG/0]      OUT npm ERR! syscall stat
2018-04-10T18:04:34.51+0200 [STG/0]      OUT npm ERR! enoent ENOENT: no such file or directory, stat '/tmp/app/.cloudfoundry/0/packages/local_modules/api-1.0.0.tgz'
```

**PACKAGE.JSON:**
```
  "dependencies": {
    "api": "file:./local_modules/api-1.0.0.tgz"
    ...
  }
```

**FOLDER:**
 - src
 - package.json
 - local_modules
     - api-1.0.0.tgz
 - ...

**Comment by Inoir** [comment]
Think same core issue: https://github.com/cloudfoundry/nodejs-buildpack/issues/133

**Comment by sclevine** [comment]
Fixed in v1.6.22.
